sourcetracker/_cli/gibbs.py

```diff
@@ -134,96 +140,76 @@ def gibbs(table_fp, mapping_fp, output_dir, loo, jobs, alpha1, alpha2, beta,
     # failed if so.
     os.mkdir(output_dir)
 
-    # Load the mapping file and biom table and remove samples which are not
-    # shared.
-    o = open(mapping_fp, 'U')
-    sample_metadata_lines = o.readlines()
-    o.close()
-
-    sample_metadata, biom_table = \
-        _cli_sync_biom_and_sample_metadata(
-            parse_mapping_file(sample_metadata_lines),
-            load_table(table_fp))
+    # Load the metadata file and feature table. Do a data check on the
+    # feature_table.
+    sample_metadata = pd.read_table(mapping_fp, index_col=0)
```

Comment:
We should centralize metadata parsing (I created #60 for this, and noted a problem with this statement there). That can happen as part of another pull request if you don't want to do it now (should be ok temporarily, but this should be fixed before the next release).